### ISSUE TITLE
(brave)  Update to v1.48.171

### ISSUE THREAD

**Issue by vairag22** [chocolatey-community/chocolatey-packages#2158]
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

1.48.171

### Download location

https://github.com/brave/brave-browser/releases/download/v1.48.171/BraveBrowserStandaloneSilentSetup.exe

### Package Page

https://community.chocolatey.org/packages/brave/1.46.153

**Comment by pauby** [comment]
Thanks for raising this issue.

The Chocolatey Community Chocolatey Packages repository is maintained by community members. To resolve this issue, there are two options:

1. Wait for a current community maintainer to fix the issue.
2. Look to fix the issue yourself, submit a pull request and engage with the community.

We have documentation that will help you:

* [Code Of Conduct](https://github.com/chocolatey-community/chocolatey-packages#etiquette-regarding-communication)
* Repository [README](https://github.com/chocolatey-community/chocolatey-packages/blob/master/README.md).
* [How to contibute to this community repository](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md).
* [Status of packages in this repository](https://gist.github.com/choco-bot/a14b1e5bfaf70839b338eb1ab7f8226f).

If you have any questions, please add a [discussion](https://github.com/chocolatey-community/chocolatey-packages/discussions/categories/q-a).

**Comment by vairag22** [comment]
Package is not updating due to [this](https://gist.github.com/choco-bot/a14b1e5bfaf70839b338eb1ab7f8226f#brave):
> Chocolatey v1.2.0 Attempting to push brave.1.48.171.nupkg to https://push.chocolatey.org Failed to process request. 'Request Entity Too Large'.  The remote server returned an error: (413) Request Entity Too Large.. Chocolatey v1.2.0 Attempting to push brave.1.49.108-beta.nupkg to https://push.chocolatey.org Failed to process request. 'Request Entity Too Large'.  The remote server returned an error: (413) Request Entity Too Large..

Seems to be an server-side issue. Not sure what I can do.

**Comment by pauby** [comment]
I haven't checked the package code s

If the package is greater than 200MB, then it would be too large to push.

The solution is to remove the x86 binary, or both binaries, to bring it down to less than 200MB. The x86 is generally less used than the x64 one which is why we recommend removing that.

**Comment by vairag22** [comment]
I see, I'll make changes to remove the x86 binaries (for beta and stable versions) and open a pull request. But I've downloaded packages that exceed 200MB limit, how does that work?

**Comment by pauby** [comment]
> But I've downloaded packages that exceed 200MB limit, how does that work?

Can you give me an example?

**Comment by vairag22** [comment]
Autodesk products, AutoCAD and such. But I think those packages probably downloaded an installer first then proceeded with rest of the installation. Nevermind.

**Comment by pauby** [comment]
Yeah, those ones do download the binary at runtime. They can't be embedded as there are no redistribution rights.

**Comment by AdmiringWorm** [comment]
A fix to partially embed this package is being pushed to Chocolatey Community Repository.

This change will still embed the 64-bit installer but will download the 32-bit installer if requested (or the running OS is 32-bit).
